.goreleaser.yml

```diff
@@ -64,11 +64,6 @@ changelog:
       - "^.*Update docs.*$"
 release:
   prerelease: auto
-  mode: prepend
-
-  header: |
-    ## Release Notes {{ .Tag }} ({{ .Date }})
-
   footer: |
     ## Thanks! the env0 team.
```

Comment:
I'd also remove the footer if we're here 😄

Reply:
@yaronya - removed.